This is due to missing implementation of do_print_to_mapfile for
atrributea section (ARM.attributes), it started to show up after fix
for PR gold/16980 was installed

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
diff --git a/gold/attributes.h b/gold/attributes.h
index c0b171f..6e3ab8c 100644
--- a/gold/attributes.h
+++ b/gold/attributes.h
@@ -387,6 +387,10 @@ class Output_attributes_section_data : public Output_section_data
   { }
 
  protected:
+  // Write to a map file.
+  void
+  do_print_to_mapfile(Mapfile* mapfile) const
+  { mapfile->print_output_data(this, _("** attributes")); }
   // Write the data to the output file.
   void
   do_write(Output_file*);
-- 
2.1.4
